exercise2.py

```diff
@@ -0,0 +1,87 @@
+import numpy as np
+import matplotlib.pyplot as plt
+
+from matplotlib.widgets import *
+
+
+class Plotter:
```

Comment:
```suggestion
class Plotter(object):
```

We need to make all our code Python2 compatible because for our ROS version we still need to use Python2 :cry: